<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editaccount</class>
 <widget class="QDialog" name="editaccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color:white;
}</string>
  </property>
  <widget class="QDateEdit" name="dateEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QDateEdit{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="locale">
    <locale language="Portuguese" country="Portugal"/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="correctionMode">
    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="timeSpec">
    <enum>Qt::UTC</enum>
   </property>
   <property name="date">
    <date>
     <year>2023</year>
     <month>11</month>
     <day>23</day>
    </date>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="boxmoney">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="suffix">
    <string>           AOA</string>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>100000000.000000000000000</double>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	background-color:rgb(59,81,127);
	color: white;
}
</string>
   </property>
   <property name="text">
    <string>CONTA-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 9pt &quot;Segoe UI &quot;;
	background-color:rgb(59,81,127);
	color: white;
}
</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linetotal">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
   <property name="placeholderText">
    <string>                                                          AOA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI&quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>√ÅREA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI </family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>QUANTIA  ARRECADADA</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI </family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>DATA PREVIOUS (DD/MM/YYYY) </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI </family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>TOTAL(AOA):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI </family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>HORA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-image: url(:/area/area/edit.png);
	background-color: rgb(72, 145, 217);
	border:1px solid white;
	background-repeat:none;
	font: 10pt &quot;Segoe UI&quot;;
	color:white;
}
QPushButton:hover{
	border:2px solid #0090d8;
}</string>
   </property>
   <property name="text">
    <string>EDITAR</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="linearea">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>211</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>211</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="maxLength">
    <number>9</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="linehour">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="inputMask">
    <string>00:00</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelperiodo">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>61</width>
     <height>30</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>61</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color:rgb(59,81,127);
	font:  11pt &quot;Segoe UI&quot;;
	color: rgb(255, 212, 38);
}
</string>
   </property>
   <property name="text">
    <string>   --</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditdate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	background-color: white;
	border:1px solid rgb(59,81,127);
	color: rgb(59,81,127);
}</string>
   </property>
   <property name="maxLength">
    <number>37</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI </family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 700 9pt &quot;Segoe UI &quot;;
	color: rgb(59,81,127);
}
</string>
   </property>
   <property name="text">
    <string>DATA CURRENT (DD/MM/YYYY) </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
